### PR TITLE
[WIP] make multiple polling handlers work

### PR DESCRIPTION
**Pull request by tkmct**
Want to add new event handlers for wallet sdk's polling.
Changed a bit of wallet.ts, PlasmaSyncher.ts, events.ts

### CODE REVIEW

**Comment by syuhei176** [comment]
I'll check!!

File: packages/wallet/src/wallet.ts [hunk 1]
```diff
@@ -234,6 +234,33 @@ export class ChamberWallet extends EventEmitter {
...
 
+  /**
+   * 
+   * @param handler 
+   * 
+   * ```typescript
+   * wallet.addUpdatedHandler((wallet) => {})
+   * ```
+   */
+  public addUpdatedHandler(handler: (wallet: ChamberWallet) => void) {
```
Comment (syuhei176) [inline packages/wallet/src/wallet.ts]: Can we use EventEmitter?

Comment (tkmct) [reply packages/wallet/src/wallet.ts]: I think event emitter is just fine.

**Comment by tkmct** [comment]
Alternatively, we use EventEmitter.
#211

**Pull request closed without merging**